php.ini-development

```diff
@@ -890,83 +893,83 @@ default_socket_timeout = 60
 ;auto_detect_line_endings = Off
 
 ;;;;;;;;;;;;;;;;;;;;;;
-; Dynamic Extensions ;
+; dynamic extensions ;
```

Comment:
I don't get these changes, this is using Title Case for "section" titles, which seems adequate?

Reply:
Yeah, I get the point. However in the curent master there is:

- ; About this file ; (First letter of the first word is uppercase)
- ; Quick Reference ; (First letter of each word is uppercase)
- ; php.ini Options  ; (First letter of the second word is uppercase – yes, I know php.ini is a file name)

What's the right convention then?